plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+/**
+ * Collects all data needed to create and configure an App Engine Standard Project.
+ */
+class AppEngineStandardProjectConfig {
+
+  private String appEngineProjectId;
+  private String eclipseProjectName;
+  private String packageName;
+  private String eclipseProjectDirectory;
+
+  // todo does builder pattern make more sense here?
```

Comment:
I expect we'll need more fields as we go forward. I want to get the basics working this quarter though.